examples/1994_06_09_aerial_scg.ipynb

```diff
@@ -0,0 +1,383 @@
GMtvbzUt+27WfCvcYFtieOQFc8Msvv5jXX3/dvP/+++bQoUNhj+3du9c8+OCDEZvb9XgvbOYGXGP7fLHZ17jcz0VCfi9cHiNsxwNwm+0+gHG3+GznB1zCtZ6bfZ3L/Zzt2svz+FgSmLgoQ3744QfTo0ePUo91Pf5UsUuWLDEJCQmmWrVqplKlSqZZs2Zm9erVoce3bNlyyo+qFbfz8Jrb9Xgvna7N/eY11vV4l2v3Gu9q7bbPF5t9je1+zhi7287mMevyGGE7HigtZfka32a862MH466b467XeJdr9xrvcu1e4129J7fd19iOd3W/uV676+NjJGLiogxZuXJlsQ9CL7Gux58qtlevXmbUqFEmOzvb7N6929x8882mZs2aZsWKFcaYku08vOZ2Od5rp2tzv9kecMrzzShtt9PX2D5fvcTb7idtbjvbx6zLY4TteKC0lOVrfJvxLo8dXuNt958uj11c59J219rOaym8llKejjmv+W0fc5GKiQuHPPXUUyf9N378+AIPQi+xrsd7zV29enXz3XffhS1LSUkx1atXN0uWLCnRzsNrbpfjvXa6Nveb7QGnPN+M0nY7fY3t89VLvO1+0ua2s33MujxG2I4H/FKer/Ftxrs8dniNt91/ujx2cZ1L211rO6+l8FpKeTrmvOa3fcxFKiYuHBIIBEz9+vVN48aN8/1Xv379Ag9CL7Gux3vNXb16dbNq1ao8yx999FGTkJBgUlNTS7Tj8prb1Xg/Om2b+83Vixyv8S7X7jXe9dptnS9+5ffS19juJ21tu0g4Zl0eI2zGA34pz9f4NuNdHju8xtvuP10fu7jOLf14l2v3Gh8JfQ2vpRQ93vX95mrtXvPbPuYiFRMXDmncuLGZNWtWgY9/8803BR6EXmJdj/eaOzk52UybNi3fxyZPnmxiYmJKrPPwmtvleK+drs39ZnvAsX2R5GrtXuNdrt3m+eJHfi/xtvtJm9vO9jHr8hhhOx7wS3m+xrcZ7/LY4TXedv/p8tjFdS5tL+14230Nr6W4eZ1cXo85r/ltH3ORiokLh1xxxRVm/PjxBT6+cuVKEwgEfI91Pd5r7pdeeslcffXVBT7+97//3TRu3LjAx710Hl5zuxzvtdO1ud9sDzjl+WaUttvpa2yfr17ibfeTNred7WPW5THCdjzgl/J8jW8z3uWxw2u87f7T5bGL61zaXtrxtvsaXktx8zq5vB5zXvPbPuYiFRMXDlmzZo1ZunRpgY8fPnzYZGZm+h7rerzX3EX1+eefm4MHD4b+Ls3O48TcLseXdqfr536zPeCU55tR2m6nr7F9vpZmvN+5bW47l445YyJrjHAtHihIeb7Gtx1fFJE0dpR2POPu/+E6l7aXdrztvobXUooX7/J+c7l2P/KXVqwf8aWFiYsyzOUTwOWL46pVq5r09PRix3vJ7zW3y/Eu7zeXzxev8S7X7jXe5dptni9+5PcSb7uftLntbB+zLo8RtuMBv9juBxh3Sz+3H/kZd90cd8vr+eY13uXavcbb7mt4LcXN6+Tyesx5zW/7mCstQaHM6tOnj37++edSj3U93mtuY0yxY73m95rb5XiX95vL54vXeJdr9xrvcu02zxc/8nuJt91P2tx2to9Zl8cI2/GAX2z3A4y7pZ/bj/yMu3Zyc77R9tKOt93X8FpK8bi831yu3Wt+28dcaWHiogxz+QRw+eLYK9v5XWV7u3G+uZfbdrzLtXtl+3x1mcv7zeXaARxjux9g3HUvt+tc3m+cb3biXa7da7ztvsZ2flfZ3m4cczgZJi4AAAAAAAAAAEDEYOICAAAAAAAAAABEDCYuAJ8FAgFnc7se72puwDW2zxebfY3L/Vwk5PfC5THCdjwAt9nuAxh33c0PuIRrPXvxrub2ynbt5Xl8LCwmLsowl08Aly+OvX7HnZf8Ln8fptd4l/eby+eL13iXa/ca73LtNs8XP/K7/L3FNred7WPW5THCdjzgF9v9AONu6ef2Iz/jrp3cnG924l2u3Wu87b6G11KKx+X95nLtXvPbPuZKSwXbBaDkuHwCuHBxbIxRTk6OoqKiwpbv2bOn2LkLm99rbtfjC1qnrdxFye93rOvxLtfuNd6F2m2fLzb7mkjs53LXayt/aR2zLo8RtuOBkubC2BWJuQsb7/rYwbjr5rhbEvEu1+413uXavca7cE9uu6+xHV/QOm3lLkp+L7G2ay+L42OpMXBeTk6OOXr0aKnHuh5f2NgjR46Ye+65x1x44YVm4sSJxhhjHnnkEVO5cmUTHR1trr32WnPo0KESye81t+vx+XFhv5VErOvxLtfuNd6F2m2fLzb7mkjs54yx29eV1jHr8hhhOx4oaS6MXZGYu7Dxro8djLtujrslEe9y7V7jXa7da7wL9+S2+xrb8flxYb95jbVde1kcH21j4sIhLp8ALl8c33vvvaZu3bpmzJgx5qyzzjJ//OMfTcOGDc2MGTPMq6++ak477TQzefLkEqnda26X413eby6fL7S9fLbd5vniR34v8bb7SZvbzvYx6/IYYTse8IvtfoBx172xw2u87f7T5bGL8422u9Z2XkvhtZTydMx5zW/7mItUTFw4xOUTwOWL4yZNmph///vfxhhjNmzYYILBoJk5c2bo8VmzZplWrVqVSO1ec7sc7/J+c/l8oe3ls+02zxc/8nuJt91P2tx2to9Zl8cI2/GAX2z3A4y77o0dXuNt958uj12cb7TdtbbzWgqvpZSnY85rftvHXKRi4sIhLp8ALl8cx8bGmk2bNoX9vW7dutDfGzduNFWrVi2R2r3mdjne5f3m8vlC28tn222eL37k9xJvu5+0ue1sH7MujxG24wG/2O4HGHf/729Xxg6v8bb7T5fHLs432u5a23kthddSytMx5zW/7WMuUjFx4RCXTwCXL47r1q1r/ve//4X+7ty5s/npp59Cf69bt85Uq1atRGr3mtvleJf3m8vni9d4l2v3Gu9y7TbPFz/ye4m33U/a3Ha2j1mXxwj
```

Comment:
The parameter "image_square_dim" can be a bit difficult to set by the user. Can it not be set automatically, based on the position of the fiducials? Or maybe it should be provided in mm, because the user may not know what should be the exact size in pixels...